src/js/badges-mvp.js

```
@@ -0,0 +1,247 @@
+/* global localStorage */
```

ESLint

[error] 'localStorage' is already defined as a built-in global variable.

(no-redeclare)